### PR TITLE
Report code coverage to coveralls.io

### PR DESCRIPTION
**Pull request by Totktonada**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,3 +14,9 @@ before_script:
 script:
   - mvn verify
   - sudo cat /var/log/tarantool/jdk-testing.log
+
+after_success:
+  - |
+    if [ "${TRAVIS_JDK_VERSION}" = openjdk11 ]; then
+        mvn clean verify jacoco:report coveralls:report -DrepoToken=${COVERALLS_TOKEN}
+    fi
```
Comment (ztarvos) [inline .travis.yml]: I maybe missing something, but I guess one can save time and collect coverage during the "script" phase and only upload results to coveralls.io on "after_success" step. Something along these lines: 
```
script:
   - mvn verify jacoco:report

after_success:
   - mvn coveralls:report -DrepoToken=${COVERALLS_TOKEN}
```

Comment (Totktonada) [reply .travis.yml]: Thanks. Fixed.

**Comment by Totktonada** [comment]
I have added badge for `connector-1.8.jdbc` branch.

**Comment by Totktonada** [comment]
@ztarvos Now looks good?

**Comment by ztarvos** [comment]
> @ztarvos Now looks good?

@Totktonada Yes, thank you.

**Pull request merged**